### ISSUE TITLE
How to find AliExpress Tracking ID?

### ISSUE THREAD

**Issue by vitorbarcelos** [alonseg/AliexpressApi#1]
Hi,

I have been trying to use the AliExpress Affiliate API. However, the routes to generate affiliate links require the Tracking ID. I have tried everywhere, API references, etc., but I have not found it.

I have been getting the following error:

`{"aliexpress_affiliate_link_generate_response":{"resp_result":{"resp_code":402,"resp_msg":"TrackingId input parameter error"},"request_id":"212a6fbf17198003803996903"}}`

Do you know where to find the TrackingId ?

**Comment by HeshamNoaman** [comment]
Yes, you can get the Tracking ID by visiting this URL: [https://portals.aliexpress.com/affiportals/web/tracking_id.htm](https://portals.aliexpress.com/affiportals/web/tracking_id.htm). Make sure you're logged into your AliExpress Affiliate account to view the Tracking ID. 
Here's a screenshot for reference:

<img width="950" alt="Screenshot 2024-12-03 160147" src="https://github.com/user-attachments/assets/74565b16-a39c-4f05-ae16-e4b1a9fdc736">
